src/assets.js

```diff
@@ -307,6 +293,20 @@ function buildAssetHTML(assetUrl, categories) {
       assetsHTML += addCategoryNamesToMixins(assetsCategoryHTML, categoryName);
     }
   }
+
+  // Iterate through catalog.json and add mixins to assetsHTML
+  catalog.forEach((item) => {
+    if (item.id && item.src) {
+      const mixinHTML = `
+        <a-mixin
+          id="${item.id}"
+          shadow
+          gltf-model="url(${item.src}/${item.id}.glb)"
+          category="${item.category || ''}"
```

Comment:
You should use `data-category` here unless you're creating a category component.

Reply:
i was following this.

```
  function addCategoryNamesToMixins(html, categoryName) {
    const parser = new DOMParser();
    const doc = parser.parseFromString(html, 'text/html');
    const mixinNodes = doc.querySelectorAll('a-mixin');
    mixinNodes.forEach((mixinNode) => {
      mixinNode.setAttribute('category', categoryName);
    });
    return doc.documentElement.innerHTML;
  }
```